trino-connector/src/main/java/org/apache/gravitino/trino/connector/system/storedprocdure/AlterCatalogStoredProcedure.java

```diff
@@ -140,7 +141,7 @@ public void alterCatalog(
     } catch (Exception e) {
       throw new TrinoException(
           GravitinoErrorCode.GRAVITINO_UNSUPPORTED_OPERATION,
-          "alter catalog failed. " + e.getMessage(),
+          "alter catalog failed. " + (StringUtils.isEmpty(e.getMessage()) ? "" : e.getMessage()),
```

Comment:
I'm not very clear about the changes here,  if `e.getMessage()` is empty or null, the code changes seems be to the same as before.

Reply:
if e.getMessage() == null. the error message is `alter catalog failed. null`. so it change it.